project.clj

```diff
@@ -48,6 +48,7 @@
                  [me.raynes/fs]
                  [liberator]
                  [org.apache.commons/commons-exec]
+                 [io.dropwizard.metrics/metrics-core "3.1.2"]
```

Comment:
Even though it's not in clj-parent currently, metrics-core probably should be added to clj-parent since it's being pinned to version 3.1.2 from multiple projects now, including tk-metrics, clj-http-client, jdbc-util, and (pe-)puppetserver.

We talked offline about this and agreed that it'd probably be easiest to just do that along with a bump to tk-metrics 1.0 once that is released.  I think @jpinsonault can take care of that as part of the work he'll be doing for [SERVER-1721](https://tickets.puppetlabs.com/browse/SERVER-1721).